### PR TITLE
WIP: DBZ-1272 Spec file for RPM package

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1272

Let's wait with merging till the spec is accepted by Fedora

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, the "description" and "installation" sections at https://copr.fedorainfracloud.org/coprs/jpechane/postgres-decoderbufs/ are empty with a scary message to users. Is this something we can influence via the RPM spec here?

**Comment by jpechane** [comment]
@gunnarmorling It is just a temprary repo but I've added the description so we can direct people there till the packages is release in Fedora.

File: rpms/postgres-decoderbufs.spec [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+License:	MIT
+URL:		https://github.com/debezium/postgres-decoderbufs
+
+%global full_version %{version}.Final
+
+Source0:	https://github.com/debezium/%{name}/archive/v%{full_version}.tar.gz
+
+BuildRequires:	gcc
+BuildRequires:	postgresql-devel >= 9.6, postgresql-server-devel >= 9.6
```
Comment (gunnarmorling) [inline rpms/postgres-decoderbufs.spec]: So on that one: we can have a single RPM which works with all PG versions starting with 9.6?

Comment (jpechane) [reply rpms/postgres-decoderbufs.spec]: @gunnarmorling Exactly

Comment (gunnarmorling) [reply rpms/postgres-decoderbufs.spec]: Ok.

**Pull request merged**